{"ID": "1e6fc13f-009d-4d47-b59b-199c03c29796", "effective_cve": "CVE-2021-3445", "image": {"exact": "docker.io/cloudbees/cloudbees-core-agent@sha256:d48f0546b4cf5ef4626136242ce302f94a42751156b7be42f4b1b75a66608880"}, "label": "TP", "package": {"name": "python3-libdnf", "version": "0.55.0-7.el8"}, "timestamp": "2023-01-26T14:03:46+00:00", "tool": "anchorectl@v1.3.0+enterprise=4.4.0+profile=rc", "user": "westonsteimel", "vulnerability_id": "CVE-2021-3445"}